pom.xml

```diff
@@ -1711,6 +1711,11 @@
                 <artifactId>guice</artifactId>
                 <version>${guice.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.google.inject.extensions</groupId>
+                <artifactId>guice-assistedinject</artifactId>
+                <version>4.0</version>
```

Comment:
It's not equals to _guice.version_ parameter?

Reply:
I used the  extension just below as  template but it does work with ${guice.version} I'll update.